.claude/hooks/pre_tool_use.py

```
@@ -0,0 +1,142 @@
+#!/usr/bin/env -S uv run --script
+# /// script
+# requires-python = ">=3.8"
+# ///
+
+import json
+import sys
+import re
+from pathlib import Path
+from utils.constants import ensure_session_log_dir
+
+def is_dangerous_rm_command(command):
```

Ruff (0.12.2)

Missing return type annotation for public function `is_dangerous_rm_command`

Add return type annotation: `bool`

(ANN201)